react-frontend/src/css/breadcrumbs.css

```diff
@@ -0,0 +1,35 @@
+.breadCrumbs {
+    position: absolute;
```

Comment:
nope

Reply:
be careful using absolute positioned element. If a breadcrumb is to be used anywhere it should not have its position dicated. Its parent component can decide its position.